config/routes.rb

```diff
@@ -104,6 +104,12 @@ def set_up_flipper
       resources :annual_reports, only: [:index, :show], param: :year do
         post :recalculate, on: :member
       end
+      resources :donations_summary, only: [:index]
+      resources :manufacturer_donations_summary, only: [:index]
+      resources :product_drives_summary, only: [:index]
+      resources :itemized_donations, only: [:index]
+      resources :itemized_distributions, only: [:index]
+      resources :activity_graph, only: [:index]
```

Comment:
I don't think I'd block the PR for this given the work already done, but these feel like separate actions on the dashboard controller, not entirely separate controllers on their own.

Reply:
dashboard controller, or a dedicated reports controller? I can refactor to a single controller with a bunch of actions if you like, no trouble.

Reply:
Naw, they are now each standalone pages / views with their own menu entries. The _used_ to be embedded in the dashboard, but with this branch they are separated completely.

Reply:
Ah I see - then yeah, maybe a reporting controller makes sense.

Reply:
Moved